### ISSUE TITLE
Colors of statistic chart are messed when change data dynamically if "Animated" is true

### ISSUE THREAD

**Issue by Mararsh** [Mararsh/MyBox#425]
This may be issue of JavaFx. Workaround is closing  "Animated" of chart.

**Comment by Mararsh** [comment]
As:
https://stackoverflow.com/questions/29124723/javafx-chart-auto-scaling-wrong-with-low-numbers?r=SearchResults
https://stackoverflow.com/questions/31774771/javafx-chart-axis-only-shows-last-label?r=SearchResults

Following lines works:
```
        histogramChart.setAnimated(false);
        histogramChart.getData().clear();

        histogramChart.setAnimated(true);
        histogramChart.getXAxis().setAnimated(false);
        histogramChart.getData().add(index, series);
```

That is:
1) Before clear/remove data, disable Animation
2) Before add data, enable Animation of chart
3) X-Axis labels are always messed when add first data, so disable Animation of X-Axis.

**Comment by Mararsh** [comment]
Fixed in v5.7
